raft/raft_test.go

```diff
@@ -348,7 +263,7 @@ func TestLeaderElectionOverwriteNewerLogs(t *testing.T) {
 func TestVoteFromAnyState(t *testing.T) {
 	vt := pb.MessageType_MsgRequestVote
 	vt_resp := pb.MessageType_MsgRequestVoteResponse
-	for st := StateType(0); st < numStates; st++ {
+	for st := StateType(0); st <= StateLeader; st++ {
```

Comment:
why change it?

Reply:
`numStates ` had been removed, this for loop is try to iterate over `StateFollover`, `StateCandidate` and `StateLeader`